modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -277,6 +298,13 @@ conf = {{{packageName}}}.Configuration(
 
     def __setattr__(self, name, value):
         object.__setattr__(self, name, value)
+        if name == 'disable_client_side_validation' and value is not None:
+            s = set(value.split(','))
+            for v in s:
+                if v not in JSON_SCHEMA_VALIDATION_KEYWORDS:
+                    raise ValueError(
```

Comment:
How about using ApiValueError so it can be captured by using our OpenApiException base exception class?